backend/app/services/container_platforms/base_platform.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod, abstractstaticmethod
+
+from app.models.experiment_template import ExperimentTemplate
+
+
+class ContainerBasePlatform(ABC):
```

Comment:
I would rename the class to `ContainerPlatformBase` and the whole module simply to `base`